Makefile

```diff
@@ -392,6 +391,13 @@ endif
 
 endif
 
+################################################################################
+# Remove overridden libraries
+################################################################################
+ifeq (${OVERRIDE_LIBC},0)
+include lib/libc/libc.mk
```

Comment:
I just realized that there are some more changes that need to be done for isolating building libraries from the assumptions in the main Makefile. We will do this separately later and apply necessary fixes to your platform as well.